.github/workflows/deploy.yml

```diff
@@ -44,6 +44,7 @@ jobs:
           echo "Deploying to: $appName"
       - name: Install package system dependencies
         run: |
+          sudo apt-get update
```

Comment:
Please stick to either `apt` or `apt-get`, the next line is using `apt`. `apt` should be the default package manager for linux distros.